tslint.json

```diff
@@ -0,0 +1,144 @@
+{
+  "rulesDirectory": [
+    "node_modules/codelyzer"
+  ],
+  "rules": {
```

Comment:
Do you really need so many rules? 
Use `"extends"` property and, for example, this [tslint-angular]( https://www.npmjs.com/package/tslint-angular) config

```
"rulesDirectory": [
    "node_modules/codelyzer"
  ],
  "extends": [
    "tslint-angular",
    "tslint:recommended"
  ],
  "rules": {
     // some custom rule
    // at first remove all and than add only you need change 
    }
}
```

Reply:
✔️